### PR TITLE
feat(jellyfish-api-core): add amount format option for listAccountHistory RPC

### PR DESCRIPTION
**Pull request by chanakasameera**
#### What this PR does / why we need it:
Configure the format of amounts returned by listAccountHistory 
id - amount with the following 'id' -> <amount>@id
symbol - amount with the following 'symbol' -> <amount>@symbol
https://github.com/DeFiCh/ain/pull/1330

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts [hunk 1]
```diff
@@ -303,6 +303,70 @@ describe('Account', () => {
...
   })
+
+  it('should listAccountHistory with options amountFormat', async () => {
+    const options = {
+      token: 'DBTC'
+    }
+    await waitForExpect(async () => {
+      const accountHistories = await client.account.listAccountHistory('mine', options)
+      expect(accountHistories.length).toBeGreaterThan(0)
+    })
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts:314]:
```suggestion
    const accountHistories = await client.account.listAccountHistory('mine', options)
    expect(accountHistories.length).toBeGreaterThan(0)
```

File: docs/node/CATEGORIES/08-account.md [hunk 2]
```diff
@@ -216,6 +216,11 @@ enum DfTxType {
   NONE = '0'
 }
 
+export enum AmountFormat {
```
Comment (surangap) [inline docs/node/CATEGORIES/08-account.md:219]:
```suggestion
enum AmountFormat {
```

**Review by canonbrother — APPROVED** [review]

**Review by surangap — COMMENTED** [review]
next fork flag

**Comment by surangap** [comment]
> next fork flag

wait for https://github.com/JellyfishSDK/jellyfish/pull/1580
it has flags set and fixed all failing tests

**Review by canonbrother — APPROVED** [review]

**Review by surangap — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**